integration-cli/docker_cli_run_test.go

```diff
@@ -106,6 +106,33 @@ func TestRunEchoStdoutWithCPUAndMemoryLimit(t *testing.T) {
 }
 
 // "test" should be printed
+func TestRunEchoStdoutWitCPUQuota(t *testing.T) {
+	defer deleteAllContainers()
```

Comment:
This probably needs a new `testRequires` to disable the test on systems that don't support this quota.
Might be good to have two tests, one that tests when the quota is supported and one for when it's not supported.... 

Someone else care to weigh in here?